pages/about.js

```diff
@@ -26,81 +33,47 @@ const AboutPage = ({ lang }) => (
     <MetaKeywords keywords={localize('about.meta-keywords', lang)} />
     <MetaImage url={DEFAULT_IMAGE} small />
 
-    <div className="wir-content-padding about-page">
-      <div className="about-page__heading">
+    <div className={cn('wir-content-padding', b())}>
+      <div className={b('heading')}>
         <Text id="about.section1-header" />
       </div>
-      <div className="about-page__text about-page__text--margin-top-small">
+      <div className={b('text', { 'margin-top': 'small' })}>
         <Text id="about.section1-text" />
       </div>
-      <div className="about-page__subheading about-page__subheading--mobile-only about-page__text--margin-top-large">
+      <div
+        className={cn(
+          b('subheading', { 'mobile-only': true }),
+          b('text', { 'margin-top': 'large' })
+        )}
+      >
         <Text id="about.section-team-header" />
       </div>
-      <div className="about-page__team">
-        <div className="about-page__subheading about-page__teammate teammate teammate__placeholder">
+      <div className={b('team')}>
+        <div className={cn(t(), t('placeholder'))}>
           <Text id="about.section-team-header" />
         </div>
         {team.map(({ image, name, role }) => (
-          <div key={name} className="about-page__teammate teammate">
-            <img className="teammate__pic" src={image} alt={name[lang] || name.be} />
-            <div className="teammate__name">{name[lang] || name.be}</div>
-            <div className="teammate__role">{role[lang] || name.be}</div>
-          </div>
+          <Teammate
+            key={image}
+            image={image}
+            name={name[lang] || name.be}
+            role={role[lang] || role.be}
+          />
         ))}
       </div>
-      <div className="about-page__text about-page__text--margin-top-medium">
-        <Text id="about.section-team-subtext">
-          {(
-            thanks,
-            name1,
-            descr1,
-            name2,
-            descr2,
-            name3,
-            descr3,
-            name4,
-            descr4,
-            name5,
-            descr5,
-            name6,
-            descr6
-          ) => (
-            <>
-              {thanks}
-              <b>{name1}</b>
-              {descr1}
-              <b>{name2}</b>
-              {descr2}
-              <b>{name3}</b>
-              {descr3}
-              <b>{name4}</b>
-              {descr4}
-              <b>{name5}</b>
-              {descr5}
-              <b>{name6}</b>
-              {descr6}
-            </>
-          )}
-        </Text>
+      <div className={b('text', { 'margin-top': 'medium' })}>
```

Comment:
will be transformed to `about_about-page__text__3Dp9k about_about-page__text--margin-top--medium__3KjEH`

Reply:
why `about_` prefix?

Reply:
doesn't matter, it's generated

imho, in the near feature, it would be just hash (`3KjEH`) in production https://github.com/zeit/next.js/issues/10339

Reply:
well I mean why it's generated like that? why doesn't it start from `about-page`

Reply:
filename?